### ISSUE TITLE
[A380X] FCU ALT ABOVE/BELOW A/C on FMA (Batch 7)

### ISSUE THREAD

**Issue by beheh** [flybywiresim/aircraft#9410]
### Description

In batch 7 two new messages have been introduced:
- FCU ALT ABOVE A/C
- FCU ALT BELOW A/C

These show when you're VS/FPA and not protected by ALT. For example, climbing with V/S and FCU ALT below current altitude would result in "FCU ALT BELOW A/C", and vice-versa.

**Comment by Pruznak** [comment]
Hello, are there any references for this ? My FCOM doesn't mention these messages (maybe because it's from 2011 lol) and I would like to take a look at this request and maybe try implementing it.

**Comment by beheh** [comment]
I have an A350 reference from the video over at https://pbnportal.eu/epbn/main/Using-PBN/PBN-on-the-Flight-Deck/RNP-Authorisation-Required--AR--Approach.html:

![A350 PFD](https://github.com/user-attachments/assets/5682521b-6983-4f8b-b25e-b68b569b46b2)

**Comment by beheh** [comment]
As far as logic goes I know it's only shown in V/S and FPA and the altitude being above/below respectively. It's to protected you from climbing/descending unrestricted.

**Comment by Pruznak** [comment]
Right, I might have a look at it. Thank you for the video and extra info!

**Comment by FoxtrotSierra6829** [comment]
> Apparently the "FCU ALT ABOVE A/C" is also only shown for 10s, whereas the other one is shown contiinuously.

The logic behind being that you can get in an underspeed condition with inadvertant +VS and with inadvertant -VS you can only CFIT? I'd consider both equally dangerous, would be interesting to know what Airbus thought if it is like this

**Comment by tracernz** [comment]
High AOA protection prevents underspeed being an issue.

**Comment by FoxtrotSierra6829** [comment]
> High AOA protection prevents underspeed being an issue.

I know the FBW protects against it in normal law, but doesn't that raise the question even more?
